frontends/election-manager/src/components/hand_marked_paper_ballot.tsx

```diff
@@ -60,6 +60,25 @@ function hasVote(
   );
 }
 
+function hasWriteInAtPosition(
+  vote: CandidateVote | undefined,
```

Comment:
I'm on the fence here as to whether we should handle undefined votes in this function or expect consumers to check that a vote is defined before calling this helper

I lean toward the latter since `hasWriteInAtPosition` is asking whether a vote has a write-in at a certain position, so it feels a bit funny to ask that question for a non-existent vote. At the same time, I see the convenience of putting the check in this function

Same question holds for for `writeInNameAtPosition`

Curious what you think!

Reply:
Ah nice, especially given that the existing `hasVote` also uses this pattern, this seems totally reasonable